src/nxdoc/licenses.md

```diff
@@ -181,7 +181,6 @@ The licenses used are:
 *   EPL 1.0: [Eclipse Public License v1.0](http://www.eclipse.org/legal/epl-v10.html)
 *   GPL 2: [GNU General Public License v2](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html)
```

Comment:
Also remove GPL 2: we don't use it. Involved libraries are dual licensed and we rely on the other one.

Reply:
will be done later: this list is not fully correct anyway